code/script.js

```diff
@@ -1,44 +1,122 @@
-
 // Step 1 - Welcome and introduction
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 );
 
-let name = prompt("What's your name?");
+let username = prompt("What's your name?");
 
-if (name != null) {
-  alert("Welcome " + name + "!")
+if (username != null) {
+  alert("Welcome " + username + "!")
   };
 
+
 // Step 2 - Food choice
-let foodChoice = prompt(
+// I found out that using parseInt is super handy with this as the user's inputs are numbers.
+// During this section I also learned that using backticks is essential to make a line break within the strings.
+
+let foodChoice = parseInt(prompt(
   `What would you like to eat today? Choose a number
   1 - Pizza
   2 - Pasta
   3 - Salad`
-);
+));
 
-switch (foodChoice) {
-
-    case "1":
-        foodChoice = alert('You chose Pizza. Good choice!');
-        break;
-    case "2":
-        foodChoice = alert('You chose Pasta. Good choice!');
-        break;
-    case "3":
-        foodChoice = alert('You chose Salad. Good choice!');
-    default:
-        foodChoice = 'not found';
-        break;
+if (foodChoice === 1) {
+  alert(`You have chosen Pizza. Good choice!`);
+} else if (foodChoice === 2) {
+  alert(`You have chosen Pasta. Good choice!`);
+} else if (foodChoice === 3) {
+  alert(`You have chosen Salad. Good choice!`);
+} else {
+  alert(`Please choose a number between 1-3 to continue.`);
 }
 
+
+
 // Step 3 - Subtype choice
-// Your code goes here
+
+let subtypeChoice;
+
+if (foodChoice === 1) {
+  subtypeChoice = parseInt(prompt(`What kind of Pizza would you like today? 
+  1 - Margherita 
+  2 - Veggie Pizza 
+  3 - White Pizza`
+  ));
+  if (subtypeChoice === 1) {
+    alert(`You have chosen Margherita.`);
+  } else if (subtypeChoice === 2) {
+    alert(`You have chosen Veggie Pizza.`);
+  } else if (subtypeChoice === 3) {
+    alert(`You have chosen White Pizza.`);
+  } else {
+    alert(`Please choose a number between 1-3 to continue.`);
+  }
+
+} else if (foodChoice === 2) {
+  subtypeChoice = parseInt(prompt(`What kind of Pasta would you like today? 
+  1 - Tomato pasta
+  2 - Pasta all'Ortollana
+  3 - Cheese pasta`
+  ));
+  if (subtypeChoice === 1) {
+  alert(`You have chosen Tomato pasta.`);
+  } else if (subtypeChoice === 2) {
+  alert(`You have chosen Pasta all'Ortollana.`);
+  } else if (subtypeChoice === 3) {
+  alert(`You have chosen Cheese pasta.`);
+  } else {
+  alert(`Please choose a number between 1-3 to continue.`);
+  }
+
+} else if (foodChoice === 3) {
+  subtypeChoice = parseInt(prompt(`What kind of Salad would you like today?
+  1 - Caesar salad
+  2 - Mediterranean salad
+  3 - Tofu salad`
+  ));
+  if (subtypeChoice === 1) {
+  alert(`You have chosen Caesar salad.`);
+  } else if (subtypeChoice === 2) {
+    alert(`You have chosen Mediterranean salad.`);
+  } else if (subtypeChoice === 3) {
+    alert(`You have chosen Tofu salad.`);
+  } else {
+    alert(`Please choose a number between 1-3 to continue.`);
+  }
+
```

Comment:
For the multiline prompts, it's recommended to either align the text with the open parenthesis or indent with a consistent number of spaces (2 or 4). Adding a comment on lines with complex logic would also help improve understanding.

Reply:
Hi Olga, 

There are many ways to structure your code, either use ESLint for Javascript code or use other similar. please find below: the YouTube channel: https://www.youtube.com/watch?v=ZuDIXV94Z1w
Once you use this app, you will know how to better structure your code. you can run this program to check your code. hope it helps. thank you

Reply:
indentation is critical in writing good code because it relates to every byte of your memory usage. the more efficient you structure your code the more faster the program would become for the user